sleap/gui/app.py

```diff
@@ -575,6 +575,13 @@ def add_submenu_choices(menu, title, options, key):
             self.commands.nextTrackFrame,
         )
 
+        add_menu_item(
+            goMenu,
+            "goto next instance change",
+            "Next Instance Change",
```

Comment:
A little ambiguous in meaning. Might I suggest
```suggestion
            "Next Instance Count Change",
```